pkg/vgmanager/vgmanager_controller.go

```diff
@@ -394,12 +399,86 @@ func (r *VGReconciler) addDevicesToVG(vgName string, devices []internal.BlockDev
 }
 
 // filterMatchingDevices returns unmatched and matched blockdevices
-// TODO: Implement this
-func filterMatchingDevices(blockDevices []internal.BlockDevice, volumeGroup *lvmv1alpha1.LVMVolumeGroup) ([]internal.BlockDevice, []internal.BlockDevice, error) {
-	// currently just match all devices
+func (r *VGReconciler) filterMatchingDevices(blockDevices []internal.BlockDevice, volumeGroup *lvmv1alpha1.LVMVolumeGroup) ([]internal.BlockDevice, []internal.BlockDevice, error) {
```

Comment:
do you plan to the use the `unmatched disks` return value? If not, then we can avoid returning that.

Reply:
no plans we can remove it

Reply:
Still see this return value in the code.

Reply:
missed it earlier, made changes now